packages/celotool/src/lib/generate_utils.ts

```diff
@@ -71,26 +71,31 @@ export const generateAccountAddressFromPrivateKey = (privateKey: string) => {
   return new Web3.modules.Eth().accounts.privateKeyToAccount(ensure0x(privateKey)).address
 }
 
+export const privateKey2GethAddress = (privateKey: string) =>
```

Comment:
nit: consider `privateKeyToStrippedAddress`, or `privateKeyToAddressNoPrefix`, or something that makes it clear that this is a normal address without the `0x` prefix